### DISCUSSION TITLE
Реализовать async и await

### DISCUSSION THREAD

**Discussion by miks1965** [pascalabcnet/pascalabcnet#2407] [General]
#445

**Comment by samuraiGH** [comment]
Как я понимаю, отсутствие async и await является основной причиной, по которой возникают проблемы подключения библиотек из C#.
В таком случае эта идея становится всё актуальнее. Уже сейчас можно проследить, как всё больше либ используют асинхронное программирование(особенно в сфере web и СУБД) и pabc.net остаётся не у дел. При таком положении язык может оказаться по сути в изоляции от .Net.

Ну это по большей части мысли в слух
